components/script_bindings/webidls/ReportBody.webidl

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// https://w3c.github.io/reporting/#interface-reporting-observer
+// TODO: Rename file to `ReportingObserver.webidl` with the other interfaces
```

Comment:
Why can you not name the file `ReportingObserver.idl` now? Does `CodegenRust.py` when the name of a file does not match any of the interfaces inside it?